.meta/transforms/filter.toml

```diff
@@ -0,0 +1,45 @@
+[transforms.filter]
+title = "Filter"
+allow_you_to_description = "select events based on a set of logical conditions"
+beta = true
+common = false
+function_category = "filter"
+input_types = ["log"]
+output_types = ["log"]
+requirements = {}
+
+<%= render("_partials/_component_options.toml", type: "transform", name: "filter") %>
+
+[transforms.filter.options.condition]
+type = "table"
+common = false
+required = true
+description = """\
+The set of logical conditions to be matched against every input event.\
+"""
+
+<%= render("_partials/_conditions.toml", namespace: "transforms.filter.options.condition.children") %>
+
+[[transforms.filter.output.log.examples]]
+label = "Hostname Matching"
+body = """\
+The `filter` transform is a simple conditional ????.
```

Comment:
Is the `????` intended?

Reply:
Um, no, oops, I though I had removed all of those placeholders.